tests/system/providers/amazon/aws/redshift/example_redshift_data_execute_sql.py

```diff
@@ -22,7 +22,11 @@
 from airflow.decorators import task
 from airflow.providers.amazon.aws.hooks.redshift_data import RedshiftDataHook
 from airflow.providers.amazon.aws.operators.redshift_data import RedshiftDataOperator
+from airflow.utils.trigger_rule import TriggerRule
+from tests.system.providers.amazon.aws.utils import set_env_id
 
+ENV_ID = set_env_id()
+DAG_ID = 'example_redshift_cluster'
```

Comment:
I can't comment below, but this should be used in the "with DAG" block:

with DAG(
    dag_id=DAG_ID,
    ....
    ) as dag:

Reply:
resolved